refactoring representation bundles

julianhyatt opened the pull request:
✨ fetch entries according to permission
✨ move permission check from glue to zed
🐛 revocation fix after update cause of missed date comparing

SpiderDan98 commented on bundles/payment-total-amount-restriction/composer.json:
```diff
@@ -0,0 +1,44 @@
+        }
+    ],
+    "require": {
+        "php": ">=7.3",
```
I guess >=7.4 would be nice for [typed properties](https://www.php.net/manual/de/migration74.new-features.php#migration74.new-features.core.typed-properties)

julianhyatt committed:
✨ fetch entries according to permission
✨ move permission check from glue to zed
🐛 revocation fix after update cause of missed date comparing

julianhyatt committed:
adjust permission logic

julianhyatt committed:
respect global permission

julianhyatt committed:
show only own entries

julianhyatt committed:
fix unit tests, phpstan and phpcs
